lib/complete.js

```diff
@@ -84,6 +97,66 @@ export default class Complete extends Domodule {
 
     return output;
   }
+
+  highlightItem() {
+    const items = toArray(this.els.resultsContainer.querySelectorAll('li'));
+
+    if (this.currentIndex < 0) {
+      this.currentIndex = 0;
+    }
+
+    if (this.currentIndex >= items.length) {
+      this.currentIndex = items.length - 1;
+    }
+
+    items.forEach(item => {
```

Comment:
just added a task for removeClass to take in array of elements, that will clean this up